### PR TITLE
Add SND_FLAGS and DROPEFFECT enums

### LINKED ISSUE #932: DoDragDrop should use DROPEFFECT enumeration
The [DoDragDrop](https://docs.microsoft.com/en-us/windows/win32/api/ole2/nf-ole2-dodragdrop) method's last two parameters are documented as containing values from [the `DROPEFFECT` enum](https://docs.microsoft.com/en-us/windows/win32/com/dropeffect-constants). The metadata should represent this.

### PR DESCRIPTION
**Pull request by AArnott**
Closes #932

### CODE REVIEW

**Comment by sotteson1** [comment]
I would expect an update to the baseline .winmd too.

**Comment by AArnott** [comment]
Agreed. I just haven't been able to build yet, with or without my changes.

File: generation/WinSDK/enums.json [hunk 1]
```diff
@@ -23131,6 +23206,37 @@
...
+    ],
+    "uses": [
+      {
+        "method": "DoDragDrop",
+        "parameter": "dwOKEffects"
+      },
+      {
+        "method": "DoDragDrop",
+        "parameter": "pdwEffect"
+      }
```
Comment (AArnott) [inline generation/WinSDK/enums.json:23237]: Add to these:

```json
      {
        "method": "SHDoDragDrop",
        "parameter": "dwEffect"
      },
      {
        "method": "SHDoDragDrop",
        "parameter": "pdwEffect"
      }
```

Comment (AArnott) [inline generation/WinSDK/enums.json:23237]: Add to these:

```json
      {
        "method": "SHDoDragDrop",
        "parameter": "dwEffect"
      },
      {
        "method": "SHDoDragDrop",
        "parameter": "pdwEffect"
      }
```

**Review by mikebattista — APPROVED** [review]

**Pull request merged**